net/switchdev-poller/Makefile

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (C) 2018-2019 Luiz Angelo Daros de Luca
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=switchdev-poller
+PKG_VERSION:=1.0.1
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Luiz Angelo Daros de Luca <luizluca@gmail.com>
+PKG_LICENSE:=GPLv2
```

Comment:
You are using outdated SPDX license. 
See https://spdx.org/licenses/

Reply:
I'll use GPL-2.0-or-later